.github/workflows/windows-ci.yaml

```diff
@@ -0,0 +1,147 @@
+name: Windows-CI
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        arch: ["x86", "x64"]
+        msvc: ["v141"]
+        qt: ["5.12.8"]
```

Comment:
Why not 5.15.0, which has a native msvc2019 target?

Reply:
Hmm... I didn't realize 5.15 was the most recent LTS. Should I drop 5.12 entirely or just add 5.15.0 to the matrix? I got a nasty surprise with 5.12.8 having x64 builds for msvc2015 and msvc2015 but not for x86. :/

Reply:
On Windows, I don't see much need for keeping 5.12, especially since the Ubuntu build will test older versions of Qt already.